Readme.md

```diff
@@ -197,10 +197,10 @@ Even cooler are the reactive extensions. Moya provides reactive extensions for
 
 ### ReactiveSwift
 
-After `ReactiveSwift` [setup](docs/ReactiveSwift.md), `request(:)` method
-immediately returns a `SignalProducer` (`RACSignal` is also available if needed)
-that you can start or bind or map or whatever you want to do. To handle errors,
-for instance, we could do the following:
+[`ReactiveSwift` extension](docs/ReactiveSwift.md) provides both `reactive.request(:callbackQueue:)` and 
+`reactive.requestWithProgress(:callbackQueue:)` methods that immediately return 
+`SignalProducer`s that you can start or bind or map or whatever you want to do. 
```

Comment:
I'd write it as `[...] that you can start, bind, map, or whatever you want to do`